packages/fxa-settings/src/pages/ResetPasswordRedesign/ConfirmResetPassword/index.test.tsx

```diff
@@ -58,9 +58,9 @@ describe('ConfirmResetPassword', () => {
 
     // name here is the accessible name from aria-label
     const submitButton = screen.getByRole('button', {
-      name: 'Submit 12345678',
+      name: 'Continue',
```

Comment:
Does this switch back to `Submit 12345678` when it's valid (or maybe you ran this by someone in a11y and they've recommended against it)?

Reply:
It doesn't currently switch back to `Submit 12345678`. I'm on the fence about this one. While including the code here can help confirm the input, including the code here was surprising/unexpected for QA (from Slack conversations) and there could be privacy concerns from having this read out loud any time the user focuses on the button.

Reply:
That's a good point about privacy concerns, though I guess one could say reading any of the numbers may elicit concerns? I still wonder what #a11y would recommend, but, I could definitely be wrong in my recommendation and it was a nice to have so I'm fine with this too.